src/Model/Changelog/Changelog.php

```diff
@@ -47,15 +47,24 @@ public function __construct(ChangelogLibrary $rootLibrary)
      * @param ChangelogLibrary $changelogLibrary
      * @return array
      */
-    protected function getLibraryLog(OutputInterface $output, ChangelogLibrary $changelogLibrary)
-    {
+    protected function getLibraryLog(
+        OutputInterface $output,
+        ChangelogLibrary $changelogLibrary,
+        ?bool $changelogAuditMode = null
```

Comment:
Why make this a tri-state instead of just default to `false`?

Reply:
Have changed to regular `bool`